draft-ietf-quic-transport.md

```diff
@@ -367,6 +367,10 @@ version negotiation and 1-RTT keys are established.
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 |                          Payload (*)                        ...
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                                                               |
++                        [FNV Hash(64)]                         +
+|                                                               |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
```

Comment:
@alagoutte as I understand it, the QUIC TLS document is currently scheduled for redesign where some details such as FNV should move to transport - @martinthomson ?